### PR TITLE
added user.data option and a toc

### PR DESCRIPTION
**Pull request by mooreds**

### CODE REVIEW

File: site/docs/v1/tech/core-concepts/users.adoc [hunk 1]
```diff
@@ -269,3 +280,19 @@ The scheduling software could know that Richard would have access to the schedul
...
 If you were to model this using only applications, you'd have to have twenty applications in FusionAuth (two for each store) and keeping those configurations synchronized might be difficult.
 And if you added more applications or stores, you'd face a combinatorial explosion of applications.
+
+=== The user.data Field
+
+You can add add arbitrary JSON to the `user.data` field, such as `org_type` and `org_id` fields. You can do this using the link:/docs/v1/tech/apis/users[User API] on user creation.
+
+This can then be read in a link:/docs/v1/tech/lambdas/jwt-populate[JWT Populate Lambda] and pushed into the tokens generated during authentication.
+
+The downstream application can examine tokens and determine to which organizations a user has access.
```
Comment (spwitt) [inline site/docs/v1/tech/core-concepts/users.adoc:290]: This read a bit awkward to me. I tried a search to find alternatives, and it turns out that there is [no rule about ending a sentence with a preposition](https://www.merriam-webster.com/grammar/prepositions-ending-a-sentence-with)!

If you had another reason for the phrasing or prefer to keep it this way, that's cool too.

File: site/docs/v1/tech/core-concepts/users.adoc [hunk 2]
```diff
@@ -152,14 +162,15 @@ include::docs/v1/tech/shared/_data-field-data-type-changes.adoc[]
...
-You have four options to do so in FusionAuth.
+You have options to do so in FusionAuth.
 They each have different tradeoffs.
 The options are:
 
 * Tenants
 * Applications and Registrations
 * Groups
 * Entities and Grants
+* The `user.data` field
```
Comment (spwitt) [inline site/docs/v1/tech/core-concepts/users.adoc:173]: Thoughts on making these links to sections?

**Review by spwitt — APPROVED** [review]
A couple of things to consider, but it looks good!

File: site/docs/v1/tech/core-concepts/users.adoc (hunk 1 above)
Comment (mooreds) [reply site/docs/v1/tech/core-concepts/users.adoc]: Awesome, didn't know that!

**Pull request merged**